pkgs/top-level/all-packages.nix

```diff
@@ -23074,6 +23074,8 @@ with pkgs;
     buildPythonApplication click future six;
   };
 
+  proselint-lsp = callPackage ../development/tools/misc/proselint-lsp {};
```

Comment:
proselint?

Reply:
that is what happens when you do 5 things at the same time, fixed, sorry about it, and thanks for pointing it out 🙏